src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,19 @@
 package core.basesyntax;
+import org.example.Ball;
+import org.example.Lottery;
 
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+
```

Comment:
Please don't add redundant empty lines to your code. We don't need them after class declaration or method signature.